src/java.base/share/classes/sun/net/www/http/HttpClient.java

```diff
@@ -98,13 +98,13 @@ private static int getDefaultPort(String proto) {
     protected int port;
 
     /* where we cache currently open, persistent connections */
-    protected static KeepAliveCache kac = new KeepAliveCache();
+    protected static final KeepAliveCache kac = new KeepAliveCache();
 
-    private static boolean keepAliveProp = true;
+    private static final boolean keepAliveProp;
 
     // retryPostProp is true by default so as to preserve behavior
     // from previous releases.
-    private static boolean retryPostProp = true;
+    private static final boolean retryPostProp;
```

Comment:
At first sight the changes to `retryPostProp` and `keepAliveProp` appear to change the default values here. However, the `static` block of this class does set these two properties to default to `true` when the corresponding system properties aren't set. So this change is fine.